addons/web/static/src/js/views/list/list_editable_renderer.js

```diff
@@ -162,7 +162,7 @@ ListRenderer.include({
      * This is a specialized version of confirmChange, meant to be called when
```

Comment:
point in commit message the commit that broke this "The 'Add a line' case is a consequence of
the recent work on list views"